src/NServiceBus.PersistenceTests/Outbox/OutboxStorageTests.cs

```diff
@@ -10,7 +10,7 @@ class OutboxStorageTests
     {
         public OutboxStorageTests(TestVariant param)
         {
-            this.param = param;
+            this.param = param.DeepCopy();
```

Comment:
Sorry I only realized it now. But we might want to make the field read only to prevent modifications, reassigning the field again at a later stage? Same for the field in the other class

Reply:
Does that matter? The parameter itself is still mutable and the only thing that can be affected is the test that is reassigning it.

Reply:
Not really. It would just be cosmetics. That's why I still approved it